dist/common/scripts/scylla_setup

```diff
@@ -62,8 +62,11 @@ def print_non_interactive_suggestion_message(args):
         print('\nAlso, to avoid the time-consuming I/O tuning you can add --no-io-setup and copy the contents of /etc/scylla.d/io*')
         print('Only do that if you are moving the files into machines with the exact same hardware')
 
+def filter_virt_netdev(netdev):
+    return os.path.exists(f'{netdev}/device') or os.path.exists(f'{netdev}/bonding')
```

Comment:
Why is bonding considered to be non-virtual device?

Reply:
Bonding is actually a virtual device, but perftune.py supports HW NIC **and bonding device**, so we should return True here.
Maybe this function name is bit misleading.